avocado/core/output.py

```diff
@@ -390,10 +390,37 @@ def early_start():
     logging.root.level = logging.DEBUG
 
 
+CONFIG = []
+
+
+def del_last_configuration():
+    if len(CONFIG) != 1:
```

Comment:
I would suggest inverting the logic here to reduce the cognitive complexity of this method:

```python
if len(config) == 1:
   return

....
```